docs/modules/deploy/pages/deploying-with-operator.adoc

```diff
@@ -132,13 +132,52 @@ You can configure the `expose-externally` field with the following properties:
 ** `NodePortNodeName`: Hazelcast clients are inside the same network as the Kubernetes nodes, so clients can access a member by the hostname of its Kubernetes node.
 ** `LoadBalancer`: Hazelcast clients are outside the Kubernetes network and can access members via the the external IP assigned to the LoadBalancer service.
 
+== Start Management Center
```

Comment:
I think this should go before the section "Connect to Hazelcast from Outside Kubernetes"

Reply:
Alright, then should I change the title to `Access the Cluster Using Management Center`?

Reply:
No, "Start Management Center" is good I guess. Why would it need to be changed?